source/js/components/nav/nav.scss

```diff
@@ -50,10 +50,8 @@
 
       &:hover,
       &:focus {
-        color: rgba(255, 237, 0, 100);
+        color: $white;
```

Comment:
This should be `$yellow`, right?

Reply:
Verified with Kristina on this. This actually fixes one of the issues mentioned in https://github.com/mozilla/foundation.mozilla.org/issues/1186